src/mme-app/structs.cpp

```diff
@@ -360,6 +365,10 @@ void Apn_name::operator = ( const Apn_name& apn_name_i )
 	memcpy( &apnname_m, &(apn_name_i.apnname_m), sizeof( apn_name_i.apnname_m ));
 }
 
+bool Apn_name::operator==(const Apn_name& apn_name_i) const
+{
+    return (memcmp(&apnname_m, &(apn_name_i.apnname_m), sizeof( apn_name_i.apnname_m )) == 0);
```

Comment:
I am not sure if we using sizeof in comparison gives correct result ? Both operands in this function initialized with 0 ?